feat: make inputs simpler (#4)
stainless-api/readme-action: README.md

## Setup

1. Copy the example from below into a GitHub workflow file (e.g. `.github/workflows/decorate.yml`)
2. Replace `INPUT_PATH` with the path to your openapi spec (relative to the root of the repo).

> 3. Replace `OUTPUT_PATH` with where you want the documented spec to be written (relative to the root of the repo). This file will be created for you.

Reviewer: ```suggestion
3. Replace `OUTPUT_PATH` with where you want the documented spec to be written (relative to the root of the repo), e.g., `my-company-openapi.documented.json`.
```
